modules/MIPS/generate_MIPS_xml.php

```diff
@@ -75,88 +66,63 @@ function get_TIN() {
 		htmlecho("ERROR:  No facility is set as the Primary Business Entity.\n   You MUST set one with the correct TIN. \n");
 		return "TINTINTINTINTINTINTIN";
 	}
-}
-
-###	==========	BEGIN MAIN	==========
-
+}?>
+
+  <p>Assumptions and instructions for generating this XML: </p>
+  <ul>
+    <li>The eligible professional has signed a waiver giving the registry permission to submit data on their behalf. <b>*REQUIRED*</b></li>
+    <li>This report must not include any "pre_" measures.</li>
+    <li>You have gone into Administration > Facilities > Mark ONE facility as 'Primary Business Entity'.  Be sure it has the correct TIN.<b> *REQUIRED*</b></li>
+    <li>Measures that must be reported on for EVERY Encounter will be manually  dealt with in the XML. </li>
+  </ul>
+<p>This tool has generated an XML file related to this report.</p>
+<p>The file naming convention is ProviderNPI-ProviderTIN.xml</p>
+<p>You may download these files by going to "<b>QA Measures</b>  -->"
+  <b> Upload Claim Files</b>" and clicking on the "<b> XML_out</b>" folder, then the appropriate <b>.xml</b> file(s).</p>
+<p>======================================================================</p>
+<?php
 echo ("<pre>\n");
-htmlecho("Assumptions and instructions for generating this XML:  \n");
```

Comment:
This stuff was originally all done with echo for purpose of code validation.  No harm changing it to straight html.